packages/contracts/test/utils/roles.utils.js

```diff
@@ -4,12 +4,16 @@ const issuerRole = ethers.utils.namehash('minter.roles.greenproof.apps.iam.ewc')
 const revokerRole = ethers.utils.namehash('revoker.roles.greenproof.apps.iam.ewc');
 const workerRole = ethers.utils.namehash('workerRole.roles.greenproof.apps.iam.ewc');
 const claimerRole = ethers.utils.namehash('claimerRole.roles.greenproof.apps.iam.ewc');
+const approverRole = ethers.utils.namehash('approver.roles.greenproof.apps.iam.ewc');
+const transferRole = ethers.utils.namehash('transferRole.roles.greenproof.apps.iam.ewc');
```

Comment:
this not needed anymore :)

Reply:
indeed !

Reply:
fixed on commit [8e3e77a](https://github.com/energywebfoundation/worker-contract/pull/186/commits/8e3e77a2c0d95101affc4fde20e0bdaee4068544)